### PR TITLE
Upgrade: Go v1.20 and packages

### PR DESCRIPTION
**Pull request by dhaiducek**
ref: https://issues.redhat.com/browse/ACM-5042

### CODE REVIEW

File: build/codebase-check.sh [hunk 1]
```diff
@@ -3,10 +3,33 @@
...
+	if [[ -n "${CI_OP_DIFF}" ]]; then
+		echo "****"
+		echo "ERROR: ${CI_OPERATOR_FILE} is not synced to $repo" | tee -a ${ERROR_FILE}
+		echo "${CI_OP_DIFF}" | sed 's/^/   /' | tee -a ${ERROR_FILE}
+		echo "***"
+		return 1
+	fi
+}
+
+# Compare the common Makefile across the repos
```
Comment (dhaiducek) [inline build/codebase-check.sh:29]: Oops. I forgot to test this. Holding to verify.

**Review by mprahl — APPROVED** [review]

**Pull request merged**